main.go

```diff
@@ -125,5 +133,62 @@ func validateOptions(opts *options) error {
 	if opts.PProfOptions.TimeSeconds < 1 {
 		return fmt.Errorf("seconds must be an integer greater than 0")
 	}
+
+	// extra FlameGraph options
+	if opts.OutputOpts.Title == "" {
+		return fmt.Errorf("FlameGraph title should not be empty")
+	}
+	if opts.OutputOpts.Width < 1200 {
+		return fmt.Errorf("FlameGraph miminal graph width is 1200 pixels")
+	}
+	if opts.OutputOpts.Colors != "" {
+		var validColors = []string{"hot", "mem", "io", "wakeup", "chain", "java", "js", "perl", "red",
```

Comment:
i think this becomes a lot simpler with a switch, and as a bonus, it's also faster.
```go
switch opts.OutputOpts.Colors {
case "hot", "mem", "io", [...]:
  // valid
default:
  // return error
}
```